### PR TITLE
DA_DP_TC065_Verify that all Pre-set Data Profiles are populated correctly

### PR DESCRIPTION
**Pull request by taile47711**

### CODE REVIEW

**Comment by taile47711** [comment]
image

**Comment by taile47711** [comment]
image

File: TADashboard/src/main/java/page_objects/DataProfilesPage.java [hunk 1]
```diff
@@ -33,4 +33,9 @@ public boolean isCheckBoxDisplayed(String profileName) {
     public void clickAddNewLink() {
         lnkAddNew.click();
     }
+
+    public boolean areDataProfilesVisible(String dataProfile) {
```
Comment (tanmle) [inline TADashboard/src/main/java/page_objects/DataProfilesPage.java]: you are just checking one data profile at a time, why do you use `are`?

**Review by tanmle — CHANGES_REQUESTED** [review]

File: TADashboard/src/main/java/page_objects/DataProfilesPage.java (hunk 1 above)
Comment (taile47711) [reply TADashboard/src/main/java/page_objects/DataProfilesPage.java]: I fixed it. Please check again

**Review by tanmle — APPROVED** [review]

File: TADashboard/src/test/java/tests/data_profiles/PositiveDPTests.java [hunk 2]
```diff
@@ -63,6 +63,34 @@ public void tc066_PreSetDataProfilesArePopulatedCorrectly() {
         }
     }
 
+    @Test(description = "Verify that all Pre-set Data Profiles are populated correctly")
+    public void tc065_VerifyThatAllPreSetDataProfilesArePopulatedCorrectly() {
+
```
Comment (MinhQuan119) [inline TADashboard/src/test/java/tests/data_profiles/PositiveDPTests.java]: Please remove

**Review by MinhQuan119 — APPROVED** [review]

**Pull request merged**